packages/gatsby/src/recipes/cli.js

```diff
@@ -0,0 +1,434 @@
+const fs = require(`fs`)
+const path = require(`path`)
+
+const React = require(`react`)
+const { useState, useContext, useEffect } = require(`react`)
+const { render, Box, Text, useInput, useApp } = require(`ink`)
+const Spinner = require(`ink-spinner`).default
+const unified = require(`unified`)
+const remarkMdx = require(`remark-mdx`)
+const remarkParse = require(`remark-parse`)
+const remarkStringify = require(`remark-stringify`)
+const jsxToJson = require(`simplified-jsx-to-json`)
+const visit = require(`unist-util-visit`)
+const humanizeList = require(`humanize-list`)
+const {
```

Comment:
This breaks users that have older version of React. We can't have two versions of react in repository, so ink version of react must match the minimal version of react we support.